src/SFA.DAS.ProviderRegistrations.UnitTests/Application/Queries/GetInvitationByIdQueryHandlerTests.cs

```diff
@@ -1,69 +1,52 @@
 ﻿using System;
 using System.Threading;
 using System.Threading.Tasks;
-using AutoMapper;
+using AutoFixture.NUnit3;
 using FluentAssertions;
-using Microsoft.EntityFrameworkCore;
-using Microsoft.EntityFrameworkCore.Diagnostics;
 using NUnit.Framework;
 using SFA.DAS.ProviderRegistrations.Application.Queries.GetInvitationByIdQuery;
 using SFA.DAS.ProviderRegistrations.Data;
-using SFA.DAS.ProviderRegistrations.Mappings;
 using SFA.DAS.ProviderRegistrations.Models;
-using SFA.DAS.ProviderRegistrations.UnitTests.Builders;
-using SFA.DAS.Testing;
 
 namespace SFA.DAS.ProviderRegistrations.UnitTests.Application.Queries
 {
     [TestFixture]
     [Parallelizable]
-    public class GetInvitationByIdQueryHandlerTests : FluentTest<GetInvitationByIdQueryHandlerTestsFixture>
+    public class GetInvitationByIdQueryHandlerTests
     {
-        [Test]
-        public Task Handle_WhenHandlingGetInvitationByIdQueryAndInvitationIsFound_ThenShouldReturnGetInvitationByIdQueryResult()
+        [Test, ProviderAutoData]
+        public async Task Handle_WhenHandlingGetInvitationByIdQueryAndInvitationIsNotFound_ThenShouldReturnNull(
+            GetInvitationByIdQueryHandler handler,
+            GetInvitationByIdQuery query)
         {
-            return RunAsync(f => f.SetInvitation(), f => f.Handle(), (f, r) => r.Should().NotBeNull()
-                .And.Match<GetInvitationByIdQueryResult>(r2 =>
-                    r2.Invitation.EmployerOrganisation == f.Invitation.EmployerOrganisation &&
-                    r2.Invitation.Ukprn == f.Invitation.Ukprn));
+            var result = await handler.Handle(query, new CancellationToken());
+            result.Should().BeNull();
         }
 
-        [Test]
-        public Task Handle_WhenHandlingGetInvitationByIdQueryAndInvitationIsNotFound_ThenShouldReturnNull()
+        [Test, ProviderAutoData]
+        public async Task Handle_WhenHandlingGetInvitationByIdQueryAndInvitationIsNotFound_ThenShouldReturnNull(
```

Comment:
`_ThenShouldReturnNull`?  

`result.Invitation.Should().NotBeNull();`? 😉

Reply:
That one was a late change, i copied the wrong previous test name to show how it is written differently. 😄